ImP/preProcessing/manipulate.py

```diff
@@ -0,0 +1,31 @@
+# IGNORE FILE
```

Comment:
I'd rename this file to something to reflect the fact that it's for prototyping.